test/dotnet.Tests/TelemetryCommandTest.cs

```diff
@@ -14,7 +14,10 @@ namespace Microsoft.DotNet.Tests
 {
     public class MockTelemetry : ITelemetry
     {
-        public string EventName{get;set;}
+        public bool Enabled { get; set; }
+
+        public string EventName { get; set; }
```

Comment:
Oh right, I see what happened.  Since you reformatted the line it showed up looking like you added a new property. I missed that this property already existed.  Sorry for the distraction.